index.js

```diff
@@ -56,7 +59,11 @@ exports.log = function(config, tags, logStatement) {
         }
         color = appColors[tag];
       }
-      tags[i] = (color) ? colors[color](tag) : colors[options.theme.tags](tag);
+      if (options.theme === false) {
```

Comment:
I think it might be cleaner to add a wrapper function that is always called and inside of that it checks if theme is false and if so just return the value otherwise, call the chalk function